src/commonmark/en/content/developer/web-api.md

```diff
@@ -13094,28 +13352,30 @@ messages.
 The Web API supports sending outgoing SMS using the POST method. SMS can
 be sent to a single or multiple destinations. One or more gateways need
 to be configured before using the service. An SMS will not be sent if
-there is no gateway configured.
- It needs a set of recipients and
+there is no gateway configured. It needs a set of recipients and
 message text in JSON format as shown below.
 
-NOTE: Recipients list will be partitioned if its size exceed
-MAX\_ALLOWED\_RECIPIENTS limit which is 200.
+    /api/33/sms/outbound
 
-    /api/26/sms/outbound
+```json
+{
+  "message":"Sms Text",
+  "recipients": [
+    "004712341234",
+    "004712341235"
+  ]
+}
+```
 
-    {
-      "message":"Sms Text",
-      "recipients": [
-        "47XXXXXX1",
-        "47XXXXXX2"
-      ]
-    }
+> ***Note***
```

Comment:
should be two asterisks each side. (I'm not sure this will render correctly)

Reply:
Fixed.